### DISCUSSION TITLE
Python 3.12 support

### DISCUSSION THREAD

**Discussion by asmunds** [Volue-Public/energy-mesh-python#420] [General]
Would it be possible to release a patch version of v1.5 that supports Python 3.12? We want to test with Python 3.12

**Comment by tnoczyns-volue** [comment]
Python 3.12 will be supported in v1.6 that's going to be released somewhere next week. Is it fine for you?

**Reply by asmunds** [reply]
Okay, next week is great!

**Reply by asmunds** [reply]
When will it be available for install via PyPi?

**Reply by tnoczyns-volue** [reply]
I'm not aware of any plans to upload Mesh Python SDK package to PyPi repository.
FYI: @torsannes

**Reply by torsannes** [reply]
There is currently no plans to make the Mesh Python SDK available via PyPi, but it might be added to future plans if we get this as a request from more users.

**Reply by tnoczyns-volue** [reply]
Mesh Python SDK v1.6 with Python 3.12 support has been released: https://github.com/Volue-Public/energy-mesh-python/releases/tag/v1.6.0.

**Reply by tnoczyns-volue** [reply]
FYI: @asmunds
